tests/api_connexion/endpoints/test_event_log_endpoint.py

```diff
@@ -291,6 +291,38 @@ def test_should_filter_eventlogs_by_when(self, create_log_model, session):
             assert response.status_code == 200
             assert {eventlog["event"] for eventlog in response.json["event_logs"]} == expected_eventlogs
 
+    def test_should_filter_eventlogs_by_included_events(self, create_log_model, session):
+        create_log_model(event="TEST_EVENT_1", when=self.default_time)
+        create_log_model(event="TEST_EVENT_2", when=self.default_time_2)
+        log_model_3 = Log(event="cli_scheduler", owner="root", extra='{"host_name": "e24b454f002a"}')
+        log_model_3.dttm = self.default_time_2
+
+        session.add(log_model_3)
+        session.flush()
```

Comment:
```suggestion
        for event in ["TEST_EVENT_1", "TEST_EVENT_2", "cli_scheduler"]:
            create_log_model(event=event, when=self.default_time)

```

Would this work instead?